static/app/components/feedback/feedbackItem/crashReportSection.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import EventOrGroupExtraDetails from 'sentry/components/eventOrGroupExtraDetails';
+import EventOrGroupHeader from 'sentry/components/eventOrGroupHeader';
+import Section from 'sentry/components/feedback/feedbackItem/feedbackItemSection';
+import {IconIssues} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Event, Group, Organization} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+interface Props {
+  crashReportId: string;
+  organization: Organization;
+  projSlug: string;
+}
+
+export default function CrashReportSection({
+  crashReportId,
+  organization,
+  projSlug,
+}: Props) {
+  const eventEndpoint = `/projects/${organization.slug}/${projSlug}/events/${crashReportId}/`;
+  const {data: crashReportData} = useApiQuery<Event>([eventEndpoint], {staleTime: 0});
+  // need to fetch the issue data since the event data is not complete enough for the component
+  const groupId = crashReportData?.groupID;
+  const issueEndpoint = `/organizations/${organization.slug}/issues/${groupId}/`;
+  const {data: groupData} = useApiQuery<Group>([issueEndpoint], {staleTime: 0});
```

Comment:
Does this mean that we'll go doing at least 3 queries?

1. eventEndpoint
2. issueEndpoint where groupdId = undefined
3. issueEndpoint where groupId = defined